docs/catalog/sinks/jira.rst

```diff
@@ -1,34 +1,39 @@
 Jira
 #################
 
-Robusta can open Jira tickets based on playbooks results.
+Robusta can open Jira tickets automatically based on issues in your cluster or Prometheus alerts
 
-Get your Jira configurations
-------------------------------------------------
 
+  .. image:: /images/jira_example.png
+    :width: 1000
+    :align: center
+    
 To configure the Jira sink you need to have following:
 
+Prerequisites
+---------------------------------
 * ``url`` : The url of your workspace. For example: https://workspace.atlassian.net (**Note:** schema (https) is required)
-* ``username`` : Jira workspace user name. For example: jira-user@company.com
-* ``api_key`` : follow the `instructions <https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/>`_ to get your api key.
-* ``project_name`` : Project for the Jira tickets.
-* ``issue_type`` : [Optional - default: ``Task``] Jira ticket type
-* ``dedups`` : [Optional - default: ``fingerprint``] Tickets deduplication parameter. By default, Only one issue per ``fingerprint`` will be created. There can be more than one value to use. Possible values are: fingerprint, cluster_name, title, node, type, source, namespace, creation_date etc
-* ``project_type_id_override`` : [Optional - default: None] If available, will override the ``project_name`` configuration
-* ``issue_type_id_override`` : [Optional - default: None] If available, will override the ``issue_type`` configuration
-
+* ``username`` : The email you use to log into your Jira account. Eg: jira-user@company.com
+* ``api_key`` : Follow these `instructions <https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/>`_ to get your api key.
+* ``project_name`` : Project you want the Jira tickets to be created in.
 
 .. note::
 
    * The configured user should have the following permissions: ``write:jira-work``, ``read:jira-work``
-   * If creating issues by ``project_name`` or ``issue_type`` fails, try specifying the corresponding ids using ``project_type_id_override`` and ``issue_type_id_override``
+
+Optional Settings
+---------------------------
+* ``issue_type`` : [Optional - default: ``Task``] Jira ticket type
+* ``dedups`` : [Optional - default: ``fingerprint``] Tickets deduplication parameter. By default, Only one issue per ``fingerprint`` will be created. There can be more than one value to use. Possible values are: fingerprint, cluster_name, title, node, type, source, namespace, creation_date etc
```

Comment:
What does it mean `There can be more than one value to use`? What does it look like if I use multiple values?